skimage/segmentation/_felzenszwalb_cy.pyx

```diff
@@ -12,50 +12,56 @@ from ..measure._ccomp cimport find_root, join_trees
 from ..util import img_as_float
 
 
-def _felzenszwalb_grey(image, double scale=1, sigma=0.8,
+def _felzenszwalb_rgb(image, double scale=1, sigma=0.8,
                        Py_ssize_t min_size=20):
-    """Felzenszwalb's efficient graph based segmentation for a single channel.
+    """Felzenszwalb's efficient graph based segmentation for 
+    single or multiple channels.
 
-    Produces an oversegmentation of a 2d image using a fast, minimum spanning
-    tree based clustering on the image grid.
+    Produces an oversegmentation of a single or multi-channel image 
+    using a fast, minimum spanning tree based clustering on the image grid.
     The number of produced segments as well as their size can only be
     controlled indirectly through ``scale``. Segment size within an image can
     vary greatly depending on local contrast.
 
     Parameters
     ----------
-    image: ndarray
+    image : ndarray
```

Comment:
Not related to this PR, but it'd be better to explicitly define supported shape:
`image : (N, M[, C]) ndarray`.